tools/issue-labeler/eng/Versions.props

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <!-- Libs -->
+    <MicrosoftAspNetCoreAllVersion>2.1.0</MicrosoftAspNetCoreAllVersion>
+    <!-- update MicrosoftDataAnalysisVersion to 2.0.0 -->
+    <MicrosoftDataAnalysisVersion>0.1.0</MicrosoftDataAnalysisVersion>
+    <OctokitVersion>10.0.0</OctokitVersion>
+    <MicrosoftAzureKeyVaultVersion>3.0.5</MicrosoftAzureKeyVaultVersion>
```

Comment:
We better not be using `Microsoft.Azure.KeyVault` anywhere; that sucker hit end-of-life in 2021 or so and is unsupported.